config/environment.js

```diff
@@ -30,6 +30,7 @@ const pollingDisabled = 'DISABLE_POLLING' in process.env
 
 const rolePermittedAnnotation = process.env.ROLE_PERMITTED_ANNOTATION || 'iam.amazonaws.com/permitted'
 const namingPermittedAnnotation = process.env.NAMING_PERMITTED_ANNOTATION || 'externalsecrets.kubernetes-client.io/permitted-key-name'
+const enforceNamespaceAnnotation = process.env.ENFORCE_NAMESPACE_ANNOTATIONS || false
```

Comment:
to make sure its always a boolean value and not; string | boolean

```suggestion
const enforceNamespaceAnnotation = 'ENFORCE_NAMESPACE_ANNOTATIONS' in process.env || false
```